src/styles/mo-components/_fixes.scss

```diff
@@ -200,3 +200,11 @@ body.donate-page {
     }
   }
 }
+
+/* Homepage join form fixes */
+.subscribe-sms-conditions {
+  display: none !important;
+}
+.subscribe-sms-conditions.unhide {
```

Comment:
I see that this might be genuinely necessary so it's not `display: block` on `.show()`/`addClass('unhide')`